data/json/items/armor.json

```diff
@@ -2781,7 +2781,7 @@
     "type": "ARMOR",
     "category": "armor",
     "name": "survivor firesuit",
-    "description": "A heavy, home-built combination armor made from a cut-down bulletproof vest and a reinforced flame-resistant jumpsuit made out of Nomex.  Protects from fire and the elements as well as from harm.",
+    "description": "Heavy, hand-built combination armor made from a cut-down bulletproof vest and a reinforced flame-resistant Nomex jumpsuit.  Protects the wearer from fire and elemental hazards.",
```

Comment:
I believe "the elements" refers to bad weathers, while "elemental harzards" refers to harzards caused by chemical elements?

Reply:
It does provide good acid resistance, but I can see how it could be confusing. Reverting.